hugo/layouts/papers/single.html

```diff
@@ -98,8 +318,39 @@ <h2 id="title">
   {{ end }}
   </p>
   </div>
-  <hr />
+  <div class="modal fade" id="metadataModal" tabindex="-1" aria-labelledby="metadataModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h5 class="modal-title">Correct Metadata for <span id="paperIdSpan"></span></h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
```

Comment:
This does not display correctly for me and also does not work. The "Export citation" modal uses this:

```html
<button class="close" data-dismiss="modal" aria-label="Close">
   <span aria-hidden="true">&times;</span>
</button>
```

Reply:
I think this might be a newer bootstrap feature, which I was using. I'll use what we know works here, though.

Reply:
I was just playing around with bootstrap for this, myself. It's responsive, but the margins don't line up. I've pushed it up as "K17-1003.html" if you want to see the static example I've been using for testing.

<img width="517" alt="image" src="https://github.com/user-attachments/assets/4a8d2f8e-69de-4edd-9680-3b7e67938e38" />

I'll be offline tomorrow so feel free to give this piece a try!